README.md

```diff
@@ -57,26 +62,28 @@ composer create-project --keep-vcs ezsystems/ezplatform-ee ezplatform ^2
+**Note:** If composer is installed locally instead of globally, the first command will start with `php composer.phar`.
+
+During the installation process you will be asked to provide database host name, login, password, etc.
+The configuration details will be placed in `<ezplatform>/app/config/parameters.yml`.
 
-During the installation process you will be asked to input things like database host name, login, password, etc.
-They will be placed in `<ezplatform>/app/config/parameters.yml`.
+Next, you will receive instructions on how to install data into the database, and how to run a simplified dev server using the `bin/console server:run` command.
 
-Next you will receive instructions on how to install data into the database, and how to run a simplified dev server using the `bin/console server:run` command.
-_Tip: For a more complete and better performing setup using Apache or Nginx, read up on how to [install eZ Platform manually](https://doc.ezplatform.com/en/latest/getting_started/install_manually/)._
+**Tip:** For a more complete and better performing setup using Apache or Nginx, see how to [install eZ Platform manually](https://doc.ezplatform.com/en/latest/getting_started/install_manually/).
 
 ## Issue tracker
-Submitting bugs, improvements and stories is possible on https://jira.ez.no/browse/EZEE.
-If you discover a security issue, please see how to responsibly report such issues on https://doc.ez.no/Security.
+Submitting bugs, improvements and stories is possible on [https://jira.ez.no/browse/EZEE](https://jira.ez.no/browse/EZEE).
+If you discover a security issue, please see how to responsibly report such issues in ["Reporting security issues in eZ Systems products"](https://doc.ezplatform.com/en/latest/guide/reporting_issues/#reporting-security-issues-in-ez-systems-products).
 
 ## Backwards compatibility
-eZ Platform aims to be **100% content compatible** with eZ Publish 5.x, 4.x and 3.x *(introduced in 2002)*, meaning that content in those versions of the CMS can be upgraded using
+eZ Platform aims to be **fully content compatible** with eZ Publish 5.x, 4.x and 3.x *(introduced in 2002)*, meaning that the content in those versions of the CMS can be upgraded using
 [online documentation](http://doc.ez.no/eZ-Publish/Upgrading) to eZ Platform.
 
-Unlike eZ Publish Platform 5.x, eZ Platform does not ship with eZ Publish Legacy (4.x). But this is available by optional installing [LegacyBridge](https://github.com/ezsystems/LegacyBridge/releases/) to allow eZ Platform and eZ Publish Legacy to run together, this is only recommended for migration use cases and not for new installations.
+Unlike eZ Publish Platform 5.x, eZ Platform does not ship with eZ Publish Legacy (4.x). But this is available by optionally installing [LegacyBridge](https://github.com/ezsystems/LegacyBridge/releases/) (available in v1.13 or lower) to allow eZ Platform and eZ Publish Legacy to run together.
```

Comment:
Note: to be kept for 2.5. In 3.0 (current master) LegacyBridge support has been dropped.